R/escape.R

```diff
@@ -229,13 +228,37 @@ sql_escape_logical <- function(con, x) {
   UseMethod("sql_escape_logical")
 }
 
+#' @keywords internal
+#' @export
+sql_escape_Date <- function(con, x) {
```

Comment:
I'd prefer to call these `sql_escape_date()` and `sql_escape_datetime()`.

Reply:
Renamed the functions.